framework/src/media/MediaRecorderImpl.cpp

```diff
@@ -358,15 +358,15 @@ int MediaRecorderImpl::getVolume()
 	return ret;
 }
 
-void MediaRecorderImpl::getRecorderVolume(int& ret)
+void MediaRecorderImpl::getRecorderVolume(uint8_t& ret)
 {
 	medvdbg("getRecorderVolume\n");
-
-	ret = get_input_audio_volume();
+	//ToDo: Add error handling code
+	get_input_audio_gain(&ret);
```

Comment:
ret or &ret?
is it correct~?

Reply:
&ret is correct. (Operations are checked.)
ret causes compile errors.